lib/styles/settings/_shadows.scss

```diff
@@ -1,6 +1,15 @@
-@import 'colors';
 @import 'colors/tokens';
 
+$color-shadow: rgba(var(--raw-black-rgb), 0.3);
+$color-success-shadow: rgba(var(--raw-green-500-rgb), 0.7);
+$color-danger-shadow: rgba(var(--raw-red-500-rgb), 0.7);
+$color-warning-shadow: rgba(var(--raw-gold-500-rgb), 0.7);
+
+// Tokens - shadows
+$color-neutrals-shadow-weak: $color-default-shadow-weak;
```

Comment:
why do we have to create an alias on this variable? Isn't original name good enough?

Reply:
I think I didn't want to touch shadows, it was before shadows migration. Now these variables have no usages, so I removed them.